### ISSUE TITLE
Percent (%) doesn’t select the closing bracket

### ISSUE THREAD

**Issue by natesilva** [aioutecism/amVim-for-VSCode#171]
I’m working through the examples on http://www.viemu.com/a-why-vi-vim.html. In example 2 (“Smart Ranges”):

```
if (!entry.used && equivalent(entry.key(), qk.key) && (curcontext & entry.contexts))
```

With the cursor in on the `e` in `equivalent`, press `c%`. It does not remove the closing parenthesis (`|` is the cursor):

```
if (!entry.used && |) && (curcontext & entry.contexts))
```

It’s supposed to do this:

```
if (!entry.used && | && (curcontext & entry.contexts)) 
```

**Comment by aioutecism** [comment]
Nice catch!
I also noticed that `%` works wired in Visual mode. 
Will fix this soon.

**Comment by aioutecism** [comment]
Fixed in 1.21.2.
PR here: #172 .
